docs/Deployment.md

```diff
@@ -0,0 +1,99 @@
+
+| Setting                  | Type     | Name                   | Description                                                                                           | Restart<br>Required |
+|--------------------------|----------|------------------------|-------------------------------------------------------------------------------------------------------|---------------------|
+| Max Partition Count      | uint16_t | MaxPartitionCount      | The maximum processor count used for partitioning work in MsQuic                                      | Yes                 |
+| Max Operations per Drain | uint8_t  | MaxOperationsPerDrain  | The maximum number of operations to drain per connection quantum                                      | No                  |
+| Retry Memory Limit       | uint16_t | RetryMemoryFraction    | The percentage of available memory usable for handshake connections before stateless retry is used    | No                  |
+| Max Worker Queue Delay   | uint32_t | MaxWorkerQueueDelayMs  | The maximum queue delay (in ms) allowed for a worker thread                                           | No                  |
+| Max Stateless Operations | uint32_t | MaxStatelessOperations | The maximum number of stateless operations that may be queued at any one time                         | No                  |
+| Initial Window Size      | uint32_t | InitialWindowPackets   | The size (in packets) of the initial congestion window for a connection                               | No                  |
+|                          |          |                        |                                                                                                       |                    |
+
+**TODO** - Finish list above
+
+## Windows
+
+On Windows, these settings can set via the registry. The main registry path for the keys is:
+
+> HKLM:\System\CurrentControlSet\Services\MsQuic\Parameters
+
+The settings can also be set per "app-name" (as indicated in `RegistrationOpen`):
+
+> HKLM:\System\CurrentControlSet\Services\MsQuic\Parameters\app-name
+
+The `DWORD` type should be used for all 32-bit or less types. For 64-bit types, `DWORD` or `QWORD` may be used. If invalid types or values are used, they will be ignored and the built-in default will be used instead.
+
+For example, to set the **Initial Window Size** setting to `20` packets, you may do the following:
+```
+reg.exe add "HKLM\System\CurrentControlSet\Services\MsQuic\Parameters" /v InitialWindowPackets /t REG_DWORD /d 20
+```
+
+# Cipher Suites
+
+## Windows
+
+By default, the new cipher suite `TLS_CHACHA20_POLY1305_SHA256` is disabled. It can be enabled via the following command:
+
+```PowerShell
+Enable-TlsCipherSuite -Name TLS_CHACHA20_POLY1305_SHA256
+```
+
+# Firewall
+
+## Windows
+
+In order to configure the Windows firewall to allow inbound QUIC traffic efficiently, use a command such as the one below.
+
```

Comment:
this should only be done when not using IIS? Please clarify when this should be done and when not.